app/search_builders/collection_search_builder.rb

```diff
@@ -9,9 +11,9 @@ def include_item_ids(solr_parameters)
     solr_parameters[:fq] << "hasCollectionMember_ssim:#{item.id}"
   end
 
-  def initialize(item, processor_chain, scope)
```

Comment:
@mjgiarlo I used the same mechanics as Hydra::Collections::SearchBehaviors instead of passing in an item to the initializer.  The only reason to override the initializer was to allow for item to be passed in.

see: https://github.com/projecthydra/hydra-collections/blob/master/lib/hydra/collections/search_behaviors.rb#L3-L5

When I added the additional method that did not use item it did not make sense to me to force a nil to be passed in.